### ISSUE TITLE
VB tuples: unexpected accessibility diagnostics for tuple members when ValueTuple type missing

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#13302]
See `GenericTupleWithoutTupleLibrary_01`

```VB

        <Fact>
        Public Sub GenericTupleWithoutTupleLibrary_01()

            Dim comp = CreateCompilationWithMscorlibAndVBRuntime(
<compilation name="NoTuples">
    <file name="a.vb"><![CDATA[
Class C
    Shared Sub Main()
        Dim x = M(Of Integer, Boolean)()
        System.Console.Write($"{x.first} {x.second}")
    End Sub
    Public Shared Function M(Of T1, T2)() As (first As T1, second As T2)
        return (Nothing, Nothing)
    End Function
End Class
]]></file>
</compilation>)

            comp.AssertTheseDiagnostics(
<errors>
BC30389: '(first As Integer, second As Boolean).first' is not accessible in this context because it is 'Private'.
        System.Console.Write($"{x.first} {x.second}")
                                ~~~~~~~
BC30389: '(first As Integer, second As Boolean).second' is not accessible in this context because it is 'Private'.
        System.Console.Write($"{x.first} {x.second}")
                                          ~~~~~~~~
BC31091: Import of type 'ValueTuple(Of ,)' from assembly or module 'NoTuples.dll' failed.
    Public Shared Function M(Of T1, T2)() As (first As T1, second As T2)
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~
BC31091: Import of type 'ValueTuple(Of ,)' from assembly or module 'NoTuples.dll' failed.
    Public Shared Function M(Of T1, T2)() As (first As T1, second As T2)
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~
BC31091: Import of type 'ValueTuple(Of ,)' from assembly or module 'NoTuples.dll' failed.
        return (Nothing, Nothing)
               ~~~~~~~~~~~~~~~~~~
</errors>)

```

**Comment by jcouv** [comment]
Fixed with #13666
